TRB_label_assist/models.py

```diff
@@ -648,29 +662,37 @@ def set_params(self, params):
 
         return self
 
-    def fit(self, train_df):
+    def fit(self, train_df,ct_entry=None):
```

Comment:
in general, if your variable is a list, it is good to reflect that in the name to make it easier to  read.
This sounds like you are passing in a single entry.

Reply:
addressed. will follow in next commit.

Reply:
This has not yet been addressed.

Reply:
Commited.